contracts/operatorfilter/opensea/OperatorFilterer.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
```

Comment:
included here for reference: this is the sample operator filterer provided by opensea. It's intended to be extended by a standard 721. This has been ported for creator in `OperatorFilter.sol`.